.github/actions/dotenv/action.yml

```diff
@@ -0,0 +1,5 @@
+name: "Setup env variables using .env file"
+description: "Load .env file from root of repo and setup for git runner"
```

Comment:
what is a "git runner"? I think this should either say "workflow runner", "CI runner" or just simply "runner"